Kickstarter-iOS/Features/ProjectPage/Controller/ProjectPageViewController.swift

```diff
@@ -644,6 +651,15 @@ public final class ProjectPageViewController: UIViewController, MessageBannerVie
     }
   }
 
+  private func goToReportProject(project _: Project) {
+    if #available(iOS 15, *) {
+      let reportProjectInfoView = ReportProjectInfoView()
+      self.viewModel.inputs.showNavigationBar(false)
```

Comment:
if you look at goToComments and goToUpdates you'll see that we tell the viewmodel to hide the nav bar before pushing those onto the stack. I think this is because the project page is being presented as a sheet. If we don't manually call `self.viewModel.inputs.showNavigationBar(false)` then the project page navbar (has the `x`, share, and heart icons) remains on pushed views. 

I'll work on getting the new view's nav title to be inline at the top, but I think this is how we need to do this for now.